dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -246,6 +246,39 @@ public void VerifyPaneDisplayModeAndDisplayModeMapping()
             });
         }
 
+        [TestMethod]
+        public void VerifyPaneDisplayModeChangingPaneAccordingly()
+        {
+            var navView = SetupNavigationView();
+
+            foreach(var paneDisplayMode in Enum.GetValues(typeof(NavigationViewPaneDisplayMode)))
+            {
+                RunOnUIThread.Execute(() =>
+                {
+                    navView.PaneDisplayMode = NavigationViewPaneDisplayMode.LeftMinimal;
+                    navView.IsPaneOpen = false;
+                    // Set it below threshold width for auto mode
+                    navView.Width = navView.CompactModeThresholdWidth - 20;
+                });
+
+                IdleSynchronizer.Wait();
```

Comment:
>IdleSynchronizer.Wait(); [](start = 16, length = 24)

Could you update this to use the new APITestBase class and the new Content Property?

Reply:
Merged branch master, now it should use APITestbBase. Thanks for pointing that out!

Reply:
So now you can use the Content property instead of needing to use the IdleSynchronizer. See some of the other controls API tests for an example

---
In reply to: [362665101](https://github.com/microsoft/microsoft-ui-xaml/pull/1788#discussion_r362665101) [](ancestors = 362665101)

Reply:
Oh I see. I will fix this tomorrow.

Reply:
Update: Removed the IdleSynchronizer.Wait and replaced it with Content.UpdateLayout in my test. 

I noticed that some tests still use the old way. Should we change these tests too (maybe in a separate PR)?

Reply:
Yeah, I did some with the initial change but some of the tests are trickier to change than others.

---
In reply to: [362741860](https://github.com/microsoft/microsoft-ui-xaml/pull/1788#discussion_r362741860) [](ancestors = 362741860)